v2/getting-started/composer.md

```diff
@@ -1,6 +1,77 @@
-# Composer: the what, why, and how
+# Using Composer
 
-@todo
+**[Composer](https://getcomposer.org/)** is a tool for managing dependencies on the project level, a project being your site or web application. It has become the de facto dependency management tool for PHP. Composer allows PHP developers to easily build standalone distributable libraries that can be shared and integrated by others. This is possible in part by the PHP Framework Interoperability Group (FIG) and [PSR-4](http://www.php-fig.org/psr/psr-4/) for autoloading of class files. 
 
-* Port contents of https://glamanate.com/blog/managing-your-drupal-project-composer
-* And https://docs.google.com/presentation/d/1PK9q2dBkGHfyEO76bEVpqS61wTgA0LGbru2PECiwUnk/edit?usp=sharing
\ No newline at end of file
+> Dependency management is not a new concept and not alone to PHP. NPM for NodeJS, Bower for front end libraries, Bundler/Gems for Ruby, PIP for Python, Maven for Java and so forth. 
```

Comment:
I kept this tidbit, because I've had a few people go "ohhhh." They were introduced to NodeJS and requiring NPM, ditto Ruby. But it didn't quite click with Composer + PHP.